_layouts/default.html

```diff
@@ -28,5 +28,13 @@
   {% include scripts.html %}
 
 </body>
-
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    tex2jax: {
+      inlineMath: [['$','$'], ['\\(','\\)']],
+      processEscapes: true
+    }
+  });
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
```

Comment:
Could you move this into scripts.html instead?  You will see it is included above.

Reply:
There is no html file with that name, would you like me to create one?

Reply:
It's here: https://github.com/byu-cpe/ecen192/blob/main/_includes/scripts.html

Reply:
Done. Thank You!